python/vyos/ifconfig/interface.py

```diff
@@ -339,7 +367,12 @@ def __init__(self, ifname, **kargs):
         self.vrrp = VRRP(ifname)
 
     def _create(self):
-        cmd = 'ip link add dev {ifname} type {type}'.format(**self.config)
+        # Do not create interface that already exist or exists in netns
+        #if self.exists(f'{self.ifname}') or get_interface_namespace(self.config['ifname']):
+        if self.exists(f'{self.ifname}') or _interface_exists_in_netns(self.config['ifname'], self.netns):
+            return
+        cmd = f'ip netns exec {self.netns} ' if self.netns else ''
+        cmd += 'ip link add dev {ifname} type {type}'.format(**self.config)
```

Comment:
I'd write it the other way arround, add the specialty later on

```python
cmd = 'ip link add dev {ifname} type {type}'.format(**self.config)
if 'netns' in self.config: cmd = f'ip netns exec {self.config["netns"]} {cmd}'
```

Reply:
Done!